src/identity/email-password-identity-provider/EmailPasswordIdentityProviderHandler.ts

```diff
@@ -0,0 +1,55 @@
+import type { IncomingMessage, ServerResponse } from 'http';
```

Comment:
The folder name can just be `src/identity/email-password/EmailPasswordIdentityProviderHandler.ts` for me.

Reply:
Cool